client/landing/stepper/declarative-flow/internals/steps-repository/store-profiler/index.tsx

```diff
@@ -79,7 +75,7 @@ const StoreProfiler: Step = function StoreProfiler( { navigation, flow } ) {
 
 	const handleSubmit = async ( event: React.FormEvent ) => {
 		event.preventDefault();
-		if ( currentUser || newUser ) {
```

Comment:
We're removing this one because the corresponding reducer will no longer be populated after removing the unused `createAccount` action, right?

Reply:
Yes, the new user is never created using the `User` store, so `newUser` is guaranteed to be `false`. This code was there either to provide interop with Gutenboarding, or, more likely, had no function at all.